src/ui/public/partials/paginated_selectable_list.html

```diff
@@ -22,7 +22,11 @@
 
     <div class="kuiBarSection">
       <p class="kuiText kuiSubduedText">
-        {{ (hits | filter: query).length }} of {{ hitCount }}
+        {{ (hits | filter: query).length }}
```

Comment:
We should pull in this also as a parameter into the string, so it becomes `{matchingHits} of {hitCount}`.